Fix missing CMake include

Authored by: ￼ Simon van der Veldt 2020-09-30
Committed by: ￼ Mark McCurry 2020-10-03

--- a/src/Nio/CMakeLists.txt
+++ b/src/Nio/CMakeLists.txt
@@ -21,6 +21,7 @@
 add_definitions(-DIN_DEFAULT="${DefaultInput}")
 
 if(JackEnable)
+    include(CheckIncludeFiles)
     include_directories(${JACK_INCLUDE_DIR})
     list(APPEND zynaddsubfx_nio_SRCS JackEngine.cpp JackMultiEngine.cpp)
     list(APPEND zynaddsubfx_nio_lib ${JACK_LIBRARIES})
